ArduCopter/APM_Config.h

```diff
@@ -59,3 +59,5 @@
 //#define USERHOOK_MEDIUMLOOP userhook_MediumLoop();        // for code to be run at 10hz
 //#define USERHOOK_SLOWLOOP userhook_SlowLoop();            // for code to be run at 3.3hz
 //#define USERHOOK_SUPERSLOWLOOP userhook_SuperSlowLoop();  // for code to be run at 1hz
+
+#define DEVO_TELEM_ENABLED ENABLED
```

Comment:
Almost is done but I can't do neither update my fork to master nor make new PR because of Revomini HAL in my master branch. So I attach patch that can be applied by
`patch -p1 <devo.patch`
in root folder
[devo.zip](https://github.com/ArduPilot/ardupilot/files/635918/devo.zip)